aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1501,7 +1533,11 @@ func updateExtendedS3Config(d *schema.ResourceData) *firehose.ExtendedS3Destinat
 
 func expandFirehoseDataFormatConversionConfiguration(l []interface{}) *firehose.DataFormatConversionConfiguration {
 	if len(l) == 0 || l[0] == nil {
-		return nil
+		// It is possible to just pass nil here, but this seems to be the
```

Comment:
We typically do not need to normalize information when sending requests to the API. This doesn't appear to hurt anything here though, so will leave this in for now.